### ISSUE TITLE
1ea59c Inconsistency in use case examples

### ISSUE THREAD

**Issue by daniel-montalvo** [act-rules/act-rules.github.io#1643]
* [Passed example 1](https://act-rules.github.io/rules/1ea59c#passed-example-1) says it has a voiceover that describes the visual information. I cannot hear any audio.
* [Failed example 2](https://act-rules.github.io/rules/1ea59c#failed-example-2) I cannot hear any audio, neither correct nor incorrect audio description.

**Comment by kengdoj** [comment]
Hi @daniel-montalvo, can you recheck the two examples that were problems? I hear audio on both.

**Comment by daniel-montalvo** [comment]
Hey @kengdoj 

I still cannot hear any audio in these two examples when I am using my windows machines. I can hear the audio, though, when using my macOS machine. Weird.

This may be something that gets fixed when merging #1680  I would say these two files that we are mentioning here are updated in that PR.

In the meantime, do you think we should leave this open until we merge #1680 ?

**Comment by kengdoj** [comment]
Weird, indeed. I'm able to hear both examples on Windows + Chrome.

> do you think we should leave this open until we merge #1680 ?

Yes, for sure.

**Comment by WilcoFiers** [comment]
#1680 should have closed this.
